tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I would use the variable <code>input_shape</code> <strong>  </strong>to set up <code>x_train</code>   to make it easier to modify.</p><p> </p><p>The name of the function<strong> </strong><code>get_gzipped_model_size</code> <strong>  </strong>seems a bit misleading to me as it has a side effect of also creating the files. I would part it into two seperate functions, one of them creating the files, and the other one returning their size.</p><p>Same with <code>setup_pretrained_weights</code> . I would do <code>train_model</code> <strong> </strong>and <code>save_weights</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks! All done.</p>